services/gitdiff/highlightdiff_test.go

```diff
@@ -21,11 +21,11 @@ func TestDiffWithHighlight(t *testing.T) {
 		"		run(db)\n",
 	)
 
-	expected := `		<span class="n">run</span><span class="o">(</span><span class="removed-code"><span class="k">&#39;</span><span class="o">&lt;</span><span class="o">&gt;</span><span class="k">&#39;</span></span><span class="o">)</span>` + "\n"
+	expected := `		<span class="n">run</span><span class="o">(</span><span class="removed-code"><span class="k">&#39;</span><span class="o">&lt;</span><span class="o">&gt;</span><span class="k">&#39;</span></span><span class="o">)</span>`
```

Comment:
Maybe you would have the problem "copy but EOL lost" again if there is no `\n` during rendering.

Reply:
Hmm yes, need to actually test this.

Reply:
Seems to copy fine in Firefox in diffs and in code view.

Reply:
Chrome is also okay, so I think this change has no impact on copy-paste.